test/unittest/custom_libcyphal_config.hpp

```diff
@@ -0,0 +1,58 @@
+/// @copyright
+/// Copyright (C) OpenCyphal Development Team  <opencyphal.org>
+/// Copyright Amazon.com Inc. or its affiliates.
+/// SPDX-License-Identifier: MIT
+
+#ifndef LIBCYPHAL_CUSTOM_LIBCYPHAL_CONFIG_HPP
+#define LIBCYPHAL_CUSTOM_LIBCYPHAL_CONFIG_HPP
+
+namespace custom
+{
+
+// Forward declaration of the custom configuration struct.
+struct MyConfig;
+
+}  // namespace custom
+
+// Overriding the default libcyphal configuration file with custom one.
+// NOLINTNEXTLINE(cppcoreguidelines-macro-usage)
+#define LIBCYPHAL_CONFIG ::custom::MyConfig
```

Comment:
Shouldn't we use a more traditional approach like in o1heap?

```c++
#ifdef LIBCYPHAL_CONFIG_HEADER
#   include LIBCYPHAL_CONFIG_HEADER
#else
#   include "libcyphal/fallback_config.hpp"
#endif
```

Reply:
ok, I'll switch gears a bit

Reply:
Actually, I forgot that one of my design goals was to give client possibility to override only needed constants while automatically inheriting other defaults as is. Please see `test/unittest/custom_libcyphal_config.hpp` where I do `struct MyConfig : libcyphal::Config { ... }`.